Makefile

```diff
@@ -3,6 +3,7 @@ VERSION = edge
 GIT_COMMIT = $(shell git rev-parse HEAD || echo "unknown")
 DATE = $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
 MANIFEST_DIR = $(shell pwd)/deploy/manifests
```

Comment:
I still think if we're going to have a `manifests/nginx-plus` directory, then we should also have a `manifests/oss` directory. A bit weird to have a structure like:

manifests/
  -  crds/
  -  services/
  - nginx-plus/
  - namespace.yaml
  - deployment.yaml

It makes it seem like nginx-plus is a sub-manifest of the oss one.

Reply:
Nevermind on this, @ciarams87 made a good suggestion that we just have a different deployment manifest.

Reply:
I commented elsewhere too, but I'm not convinced we need two separate folders, especially for CRDs, it seems confusing to me. Can't we just have an extra deployment manifests and keep the rest as is? Or optionally create one new folder called `nginx-gateway/` (or something else)

So, something simply like:

manifests/
- crds/
- services/
- namespace.yaml
- nginx-gateway.yaml
- nginx-plus-gateway.yaml

Or 
manifests/
- crds/
- services/
- namespace.yaml
- nginx-gateway/
    - nginx-gateway.yaml
    - nginx-plus-gateway.yaml

Services also has multiple manifests when you only actually deploy one, so it fits the pattern

Reply:
I see benefits both ways, not sure which one I prefer.